app/models/word.rb

```diff
@@ -1,7 +1,41 @@
 class Word
   include Mongoid::Document
-  field :word_name, type: String
+  include Mongoid::Attributes::Dynamic
+
+
+  belongs_to :syntactical_category, class_name: "Category", inverse_of: :syntactical_words, :autosave => true
+  has_and_belongs_to_many :semantical_categories, class_name: "Category", inverse_of: :semantical_words, :autosave => true
+
+  field :name, type: String
   field :description, type: String
-  field :syntactical_category, type: String
-  field :semantical-categories, type: String
+
+
+  # OPTIMIZE: if the word allready exist, it should ask the user if he wants to edit the existing word.
+  validates :name, uniqueness: {message: "Dieses Wort gibt es bereits." }
```

Comment:
Please internationalize. You save yourself some hassle if you do this from the start and never use localized text in the code.